opendatahub-io/manifests
# Integrating RH Service Mesh with Kubeflow

This Pull request will include all the necessary changes needed to integrate RH Mesh Service (installed from Operator Catalog) with Kubeflow 0.7
Solving Issue: https://gitlab.com/opendatahub/opendatahub-operator/issues/116
To test:
1. Install RH Service Mesh by following instructions: https://access.redhat.com/documentation/en-us/openshift_container_platform/4.2/html/service_mesh/service-mesh-installation#installing-ossm. Make sure to wait for each operator install to be successful before moving to the next step. 
2. git clone the kubeflow manifest repo
3. build and apply with config file kfctl_openshift-servicemesh.yaml
3. For testing if the integration succeeded
     a. oc get virtualservices
     b. Access the portals using http://<istio ingress url>/<path of component from virtualservice>
4. Also we can test by launching a notebook server in a specific namespace. However this namespace has to be added to the Service Mesh Member Roll instance called "default" in the istio-system namespace.
